index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
+        <nav class="menu-icon">
+            <h4 class="logo"><a href="#">My Logo</a></h4>
+            <img class="menubars" src="menu.png" alt="Menu"> 
+        </nav>
+        <section class="homepage">
+            <div class="intro">
+                <h2>
+                    I'm Abu Talha. <br>  
+                    Glad to see <br> you!
+                </h2>                                  
+            </div>
+            <div class="detail">
+                <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>    
+            </div>
+            <div class="connect">
+                <h3>LET'S CONNECT</h3>
+                <div class="logos">
+                    <ul>
+                        <li><a href=""><img src="Vector.png" alt="Twitter"></a></li>
+                        <li><a href=""><img src="Linkedin icon.png" alt="Linked in"></a></li> 
+                        <li><a href=""><img src="msg.png" alt="message"></a></li>
+                        <li><a href=""><img src="github.png" alt="github"></a></li>
+                        <li><a href=""><img src="victory.png" alt="victory"></a></li>                       
+                    </ul>
+                    <!-- <img src="Vector.png" alt="Twitter">
+                    <img src="Linkedin icon.png" alt="Linked in">
+                    <img src="msg.png" alt="message">
+                    <img src="github.png" alt="github">
+                    <img src="victory.png" alt="victory">   -->
```

Comment:
- [ ] You may consider deleting the commented lines of code to stick to the best practice as it is the standard and because they make your code ugly and hard to read.